configure.ac

```diff
@@ -46,5 +46,12 @@ AC_TRY_LINK([
 AC_DEFINE(HAVE_FULL_QUEUE_H,, Define if sys/queue.h includes required macros)
 ])
 
+AC_TRY_LINK([
+	#include <ncursesw/curses.h>
+	#include <ncursesw/term.h>
+], [], [
+AC_DEFINE(HAVE_NCURSESW_H,, Define if ncursesw is available)
+])
```

Comment:
I'm sorry if I'm not getting this correctly, but won't `AC_SEARCH_LIBS` define `HAVE_NCURSESW_H` in `config.h` if `ncursesw` exists, that will then be available in `src/ui.c`?

What I'm suggesting is we drop `AC_TRY_LINK`. Does that build on your system?